pkg/vere/main.c

```diff
@@ -1170,13 +1175,23 @@ _cw_eval(c3_i argc, c3_c* argv[])
       case c3__loom: {
         c3_w lom_w;
         c3_o res_o = _main_readw(optarg, u3a_bits + 3, &lom_w);
-        if ( (c3n == res_o) || (lom_w < 20) ) {
-          fprintf(stderr, "error: --loom must be >= 20 and <= %u\r\n", u3a_bits + 2);
+        if ( (c3n == res_o) || (lom_w < 24) ) {
+          fprintf(stderr, "error: --loom must be >= 24 and <= %u\r\n", u3a_bits + 2);
```

Comment:
re: usage of `u3a_bits`. I actually prefer a literal `32` here. The maximum bex of a loom really doesn't have anything to do with `u3a_bits` which just denotes the number of bits in a noun that address the loom.

-- Though it is the pattern followed elsewhere. So probably don't worry about it and I'll just fix it like 51edb14c582817e73069c24c14fa3567e0e2aad0 whenever #164 gets merged into develop